bin/initMDSPdo.php

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
+<?php
+
+/* This is the base directory of the simpleSAMLphp installation. */
+$baseDir = dirname(dirname(__FILE__));
+
+/* Add library autoloader and configuration. */
+require_once $baseDir . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . '_autoload.php';
+require_once $baseDir . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'config.php';
+
+echo "Initializing Metadata Database...". PHP_EOL;
+
+# Iterate through configured metadata sources and ensure
+# that a PDO source exists.
+foreach ($config['metadata.sources'] as $source) {
+	# If pdo is configured, create the new handler and initialize the DB.
+	if ($source['type'] === "pdo") {
+		$metadataStorageHandler = new SimpleSAML_Metadata_MetaDataStorageHandlerPdo($source);
+		$result = $metadataStorageHandler->initDatabase();
+
+		if ($result === FALSE) {
```

Comment:
We're also trying to get rid of all appearances of `TRUE`, `FALSE`, and `NULL` in uppercase.